src/ui/public/notify/notifier.js

```diff
@@ -318,48 +320,69 @@ Notifier.prototype._showFatal = function (err) {
  * @param  {Error|String} err
  * @param  {Function} cb
  */
-Notifier.prototype.error = function (err, cb) {
-  return add({
+Notifier.prototype.error = function (err, opts, cb) {
+  const overrideable = ['lifetime', 'icon'];
+  if (_.isFunction(opts)) {
```

Comment:
Is there anywhere in the code that actually passes a callback to `.error`? If not, or if there aren't many, this code wouldn't be needed